otx/core/ov/ops/op.py

```diff
@@ -32,7 +32,7 @@ class Operation(torch.nn.Module, Generic[_T]):  # pylint: disable=abstract-metho
     """Operation class."""
 
     TYPE = ""
-    VERSION = -1
+    VERSION = ""
```

Comment:
Just curious, what is the meaning of VERSION?

Reply:
It represents opset in terms of openvino: opset1, opset2, etc. Some of ops still have integer opset values. You may manually inspect openvino.xml you'll find it.
Unfortunately, now we have no resources to update all the ops with the correct opset versions, but I did that for the ones, that are used in tests.